File: 2013-06-07/python/exercise1.py
from pyplasm import *
from numpy import *

dom1D = INTERVALS(50)(32)
domain = PROD([dom1D,dom1D])

def mounts(dom):
	x = dom[0];
	y = dom[1];
	z = random.random();
 	return [x, y, z];

terrain = MAP(mounts)(domain);

VIEW(terrain);

File: 2013-06-07/javascript/exercise1.js
var domain = DOMAIN([[-50, 50], [-50, 50]])([24, 24]);

var mounts = function(dom, dz){
    var x = dom[0];
    var y = dom[1];
    var z = Math.random()*Math.abs(x%y);
    if (z==0)
    	z = Math.random();
    return [x, y, z];
}

var terrain = MAP(mounts)(domain);

DRAW(terrain);